plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -155,3 +156,22 @@ def test_nevergrad_example(with_commandline: bool, tmpdir: Path) -> None:
     # check that all job folders are created
     last_job = max(int(fp.name) for fp in Path(tmpdir).iterdir() if fp.name.isdigit())
     assert last_job == budget - 1
+
+
+@pytest.mark.parametrize("max_failure_rate", (0.5, 1.0))
+def test_failure_rate(max_failure_rate: float, tmpdir: Path) -> None:
+    cmd = [
+        "example/my_app.py",
+        "-m",
+        f"hydra.sweep.dir={tmpdir}",
+        "hydra.sweeper.optim.budget=2",  # small budget to test fast
+        "hydra.sweeper.optim.num_workers=2",
+        f"hydra.sweeper.optim.max_failure_rate={max_failure_rate}",
+        "error=true",
+    ]
+    if max_failure_rate >= 1.0:
+        # nevergrad will warn that the values are too high (infinity)
+        run_python_script(cmd, allow_warnings=True)
+    else:
+        with pytest.raises(subprocess.CalledProcessError):
+            run_python_script(cmd)
```

Comment:
Could we assert the exception and the warning?

maybe something similar [here](https://github.com/facebookresearch/hydra/blob/6349706f4413c9431bd1d5bfcabb3ae041a33a4d/tests/test_basic_sweeper.py#L62) where we assert traceback in the test for BasicSweeper.

Reply:
I've updated the test, is it better ?